src/Abp.AutoMapper/AutoMapper/AutoMapAttribute.cs

```diff
@@ -21,8 +21,8 @@ public override void CreateMap(IMapperConfigurationExpression configuration, Typ
 
             foreach (var targetType in TargetTypes)
```

Comment:
@naazz1992 should we remove the foreach loop here ?

Reply:
Yes we should for sure. I forget to remove it.